StSpinPool/StFmsTriggerMaker fails to compile with ROOT6

See #63 

The error message is:

https://github.com/star-bnl/star-sw/pull/63/checks?check_run_id=3108542102#step:3:11725

```
#14 1229.7 .sl88_gcc789/OBJ/StRoot/StSpinPool/StFmsTriggerMaker/StFmsTriggerMaker.h:77:32: error: use of undeclared identifier 'MAXPP'
#14 1229.7   int FmsLargeClusterTh1(int t=MAXPP) const { return btest(fp201.output[t],6); }
#14 1229.7                                ^
#14 1229.7 .sl88_gcc789/OBJ/StRoot/StSpinPool/StFmsTriggerMaker/StFmsTriggerMaker.h:78:32: error: use of undeclared identifier 'MAXPP'
#14 1229.7   int FmsLargeClusterTh2(int t=MAXPP) const { return btest(fp201.output[t],7); }
#14 1229.7                                ^
#14 1229.7 fatal error: too many errors emitted, stopping now [-ferror-limit=]
#14 1229.7 Error: rootcint: compilation failure (.sl88_gcc789/OBJ/StRoot/StSpinPool/StFmsTriggerMaker/StFmsTriggerMaker_Cintf7f9bbde89_dictUmbrella.h)
#14 1229.7 cons: *** [.sl88_gcc789/OBJ/StRoot/StSpinPool/StFmsTriggerMaker/StFmsTriggerMaker_Cint.cxx] Error 2
#14 1229.8 cons: errors constructing .sl88_gcc789/OBJ/StRoot/StSpinPool/StFmsTriggerMaker/StFmsTriggerMaker_Cint.h
```

To reproduce with the container do:

```bash
git clone https://github.com/star-bnl/star-sw.git
cd star-sw
git checkout ff4e30692d560f91719fe83ea3be2b13260a7133
docker buildx build --cache-from=type=registry,ref=ghcr.io/star-bnl/star-sw-base-root6@sha256:e52132bc193a45271b43f2631c39bfc7a3c6aa26f2766cccd7ee6d645d58b5a8 -f docker/Dockerfile.root6 .
```

--- a/StRoot/StSpinPool/StFmsTriggerMaker/StFmsTriggerMaker.h
+++ b/StRoot/StSpinPool/StFmsTriggerMaker/StFmsTriggerMaker.h
@@ -13,7 +13,7 @@
 class TH2F;
 
 #include <cstring>
-#if !defined(__CINT__) && !defined(__CLING__)
+#if !defined(__CINT__)
 #include "Board.hh"
 #include "Crate.hh"
 #include "bits.hh"
@@ -131,7 +131,7 @@ private:
 
   // Crates
   enum { NCRATES = 14 };
-#if !defined(__CINT__) && !defined(__CLING__)
+#if !defined(__CINT__)
 
   Crate crates[NCRATES];
 

--- a/docker/Dockerfile.root6
+++ b/docker/Dockerfile.root6
@@ -92,6 +92,6 @@ RUN source star-spack/setup.sh && spack env activate star-env-root6 \
  && cons %pams/sim/g2r %OnlTools \
          %StarVMC/GeoTestMaker \
          %StRoot/StarGenerator/Kinematics \
-         %StRoot/StEEmcPool %StRoot/StFgtPool %StRoot/StHighptPool %StRoot/StSpinPool \
+         %StRoot/StEEmcPool %StRoot/StFgtPool %StRoot/StHighptPool \
          %StRoot/StJetFinder \
          %StRoot/StShadowMaker %StRoot/StSpinMaker %StRoot/StHbtMaker

